src/libraries/System.Collections/tests/BitArray/BitArray_CtorTests.cs

```diff
@@ -66,6 +66,17 @@ public static void Ctor_Int_Bool(int length, bool defaultValue)
             Assert.False(collection.IsSynchronized);
         }
 
+        [Fact]
+        public static void Ctor_Int_Bool_ShouldNotLeaveDirtyBits()
+        {
+            BitArray bitArray = new BitArray(33, true);
```

Comment:
Given that we're indexing into the underlying array's last element, consider adding a test for `new BitArray(0, true)` as well.

Reply:
Nevermind, I see we already have one above.